common/css-paint-tests.js

```diff
@@ -0,0 +1,23 @@
+// We call rAF and in the second frame we compare the result with the ref.html.
+function importPaintWorkletAndTerminateTestAfterAsyncPaint(code) {
+    if (window.testRunner) {
+        testRunner.waitUntilDone();
+    }
+    var blob = new Blob([code], {type: 'text/javascript'});
+    var frame_cnt = 0;
+    paintWorklet.addModule(URL.createObjectURL(blob)).then(function() {
+        requestAnimationFrame(function() {
+            takeScreenshot(frame_cnt);
+        });
+    });
+}
+
+function takeScreenshot(frame_cnt) {
```

Comment:
How about introducing a `waitForAnimationFrames(n)` or similar, and using, `waitForAnimationFrames(n).then(takeScreenshot)`, where `takeScreenshot` comes from common/reftest-wait.js?